src/OptionSettlement.sol

```diff
@@ -155,9 +169,8 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
             revert InvalidAssets(optionInfo.exerciseAsset, optionInfo.underlyingAsset);
         }
 
-        // Use the optionKey to seed entropy
-        optionInfo.settlementSeed = optionKey;
```

Comment:
What are we replacing this with?

Reply:
I'm re-adding it, per your other comment